README.md

```diff
@@ -6,14 +6,14 @@ repository](https://github.com/unison-code/unison).
 
 ## Building, Testing, and Installing
 
-Just checkout one of the branches with a `-unison` suffix (corresponding to
-upstream LLVM's master and release branches) and follow the instructions
+Just checkout `release_38-unison`(corresponding to
```

Comment:
Since there is essentially only one usable branch, why not just write "Just checkout the branch `release_38-unison` and follow the instructions..."?

Reply:
Yea, true! I have fixed that.